### PR TITLE
STU3: cleaning up solution

### PR DESCRIPTION
**Pull request by marcovisserFurore**
- updated the download script for the downloading the FHIR specification
- .xsd and .sch files are updated to version 3.0.1

### CODE REVIEW

File: src/Hl7.Fhir.Serialization.Tests/TestData/json-edge-cases.json [hunk 1]
```diff
@@ -1,201 +1,168 @@
 {
```
Comment (ewoutkramer) [inline src/Hl7.Fhir.Serialization.Tests/TestData/json-edge-cases.json:1]: Do you have an idea why this file has changed so much?  Is it just whitespace (or I seem some encoding of &nbsp; to 0x202f?

File: src/Hl7.Fhir.Specification/data/account.xsd [hunk 2]
```diff
@@ -27,7 +27,7 @@
   POSSIBILITY OF SUCH DAMAGE.
   
 
-  Generated on Tue, Mar 21, 2017 06:09+1100 for FHIR v3.0.0 
+  Generated on Wed, Apr 19, 2017 07:44+1000 for FHIR v3.0.1
```
Comment (ewoutkramer) [inline src/Hl7.Fhir.Specification/data/account.xsd:30]: Ah we shipped the pre-technical correction XSDs....no one ever complained ;-)

**Review by ewoutkramer — APPROVED** [review]

File: src/Hl7.Fhir.Specification/data/account.xsd (hunk 2 above)
Comment (marcovisserFurore) [reply src/Hl7.Fhir.Specification/data/account.xsd:30]: Yes, indeed. I also noticed this, when I was doing the downloads again

File: src/Hl7.Fhir.Serialization.Tests/TestData/json-edge-cases.json (hunk 1 above)
Comment (marcovisserFurore) [reply src/Hl7.Fhir.Serialization.Tests/TestData/json-edge-cases.json:1]: It is mostly white space. For example:
it was:
```
"identifier": [
        {
          "system": "urn:ietf:rfc:3986",
          "value": "2.16.840.1.113883.19.5"
        }
      ],
```
and now it is:
```
"identifier": [{
                "system": "urn:ietf:rfc:3986",
                "value": "2.16.840.1.113883.19.5"
            }],
```

But during the download, something went wrong, I see now. The family name of the contact should be `du Marché`, but is now `du MarchÃ©`. So I think the encoding is not right in the download phase. I will have a look at it.
